gravitee-apim-e2e/ui-test/support/PageObjects/Apis/ApiDetails.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// gio-submenu-item__title and gio-submenu-item for Design
+
+class apiDetails {
+  policyStudio() {
+    return cy.get('.gio-submenu-item').contains('Policy Studio');
+  }
+
+  info() {
+    return cy.get('.gio-submenu-item__title').contains('Info');
+  }
+
+  plans() {
+    return cy.get('.gio-submenu-item__title').contains('Plans');
+  }
+}
```

Comment:
The method names are not very clear to me, reading `ApiDetails.info()` in the code did not indicate me it was an entry in the menu.

Maybe we can choose more meaningful names, like `getPolicyStudioMenuItem()`.

Also, can you indicate the return type of the method in the signature please ?